django/cantusdb_project/main_app/models/institution.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+from main_app.models import BaseModel
+
+
+region_help_text = """Province / State / Canton / County. Used to disambiguate cities, e.g., "London (Ontario)"."""
+city_help_text = """City / Town / Village / Settlement"""
+
+
+class Institution(BaseModel):
+    name = models.CharField(max_length=255, default="s.n.")
+    siglum = models.CharField(max_length=32, default="XX-Nn")
+    city = models.CharField(
+        max_length=64, blank=True, null=True, help_text=city_help_text
+    )
+    region = models.CharField(
+        max_length=64, blank=True, null=True, help_text=region_help_text
+    )
+    country = models.CharField(max_length=64, default="s.l.")
+    alternate_names = models.TextField(
+        blank=True, null=True, help_text="Enter alternate names on separate lines."
+    )
+    former_sigla = models.TextField(
+        blank=True, null=True, help_text="Enter former sigla on separate lines."
+    )
+
+    def __str__(self) -> str:
+        names: list = [self.name]
+        if self.siglum:
+            names.append(f"({self.siglum})")
+        return " ".join(names)
```

Comment:
Isn't this just
```suggestion
        return f"{self.name} ({self.siglum})"
```

Reply:
The original checks if self.siglum is null and appends it to the list with the parens. In your version if it’s null you’ll get empty parens, or maybe a parenthesised “None”.

Reply:
Yes...but self.siglum is a non-nullable field with a default value, right?

Reply:
Yeah, that's true. But I guess I'm also hedging my bets that at some point it may not be a requirement, and I don't want this bit of code to break.

Reply:
Fixed.